tests/modules/nf-core/modkit/pileup/main.nf

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env nextflow
```

Comment:
This seems to be related to Pytest. We exclusively use nf-test now, so there is no need for these files.

Reply:
Good spot!